<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1291</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PRODUCTO</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>521</width>
        <height>268</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CODIGO</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="insertarCodigo">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PRECIO</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="insertarPrecio">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOMBRE</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="insertarNombre">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DESCRIPCION</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="insertarDescripcion">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="insertarBoton">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>AGREGAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INSERTAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MODIFICAR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxEditar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="editarBoton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EDITAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ELIMINAR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxEliminar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="borrarBoton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ELIMINAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>550</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CARGAR</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>691</width>
        <height>641</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>TABLA</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>1</y>
          <width>691</width>
          <height>611</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>CODIGO</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PRECIO</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NOMBRE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>DESCRIPCION</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>GRAFICO</string>
       </attribute>
       <widget class="QScrollArea" name="scrollAreaP">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>611</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>689</width>
           <height>609</height>
          </rect>
         </property>
         <widget class="QLabel" name="repo">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>681</width>
            <height>611</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>550</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GENERAR GRAFO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>CLIENTE</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>621</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_43">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NIT</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="insertarNit">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_44">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOMBRE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="insertarNombreC">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_45">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>COMPRAS</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="insertarComprar">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_11">
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>AGREGAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INSERTAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MODIFICAR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboEditarCliente">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="buscarCliente">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>300</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EDITAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>370</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ELIMINAR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboEliminarCliente">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminarCliente">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>430</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ELIMINAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cargarCliente">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>540</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CARGAR</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>621</width>
        <height>651</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_27">
       <attribute name="title">
        <string>TABLA</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>1</y>
          <width>571</width>
          <height>591</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>NIT</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NOMBRE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>COMPRAS</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_28">
       <attribute name="title">
        <string>GRAFICO</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>FACTURA</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1297</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
